python/jupytergis_lab/jupytergis_lab/notebook/geo_debug.py

```diff
@@ -0,0 +1,33 @@
+from sidecar import Sidecar
+
+from jupytergis_lab import GISDocument
+
+
+def geo_debug(geojson_path: str) -> None:
+    """Run a JupyterGIS data interaction interface alongside a Notebook."""
+    # TODO: allow user to specify a different project file;
+    # TODO: Just create the .jgis file
+    doc = GISDocument("debug.jGIS")
```

Comment:
If you make it nameless, IIRC that will make it fully in-memory and not save a file on the side. This resolves one of your concerns in the associated issue.